src/styles/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+.page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stopwatch {
+  @include circle($stopwatch-size);
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1vmin dotted #000;
+
+  &__minutes-hand {
+    height: $minutes-hand-height;
+    width: $minutes-hand-weigth;
+    background-color: $minutes-hand-color;
+    transform-origin: bottom center;
+    bottom: 50%;
+    position: absolute;
+    z-index: 0;
+    animation: rotate $min-animation-time steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    height: $seconds-hand-height;
+    width: $seconds-hand-weigth;
+    background-color: $seconds-hand-color;
+    position: absolute;
+    z-index: 1;
+    transform-origin: bottom center;
+    bottom: 50%;
+    animation: rotate $sec-animation-time linear infinite;
+
+  }
+  &__center {
+    @include circle($center-size);
+
+    background-color: $center-color;
+    position: absolute;
+    z-index: 2;
+
+  }
+
+  &--speed-up {
+    .stopwatch__minutes-hand {
+      animation: rotate $min-animation-time-speed-up steps(60) infinite;
+    }
+
+    .stopwatch__seconds-hand {
+      animation: rotate $sec-animation-time-speed-up linear infinite;
+    }
+  }
+}
```

Comment:
add modifier inside minutes and seconds class [Typical BEM Mistakes](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#increasing-modifier-specificity)

Reply:
Przepraszam, ale czy możesz mi powiedzieć, jak to zrobić poprawnie?
  Ponieważ tutaj wartości animacji dla testów z klasą stopwatch--speed-up zostały po prostu zmienione.
  Ale nie rozumiem, jak nadać animacji inne wartości, pod warunkiem, że istnieje klasa stopwatch--speed-up bezpośrednio w w stopwatch__minutes-hand i stopwatch__seconds-hand?